internal/service/database/database.go

```diff
@@ -375,8 +351,28 @@ func waitServiceNameToPropagate(ctx context.Context, name string) (err error) {
 	return errors.New("max retries reached while waiting for service name to propagate")
 }
 
-func updateDatabaseVersion(ctx context.Context, d *schema.ResourceData, client *service.Service) diag.Diagnostics {
-	diags := diag.Diagnostics{}
+func updateDatabaseVersion(ctx context.Context, d *schema.ResourceData, client *service.Service) (diags diag.Diagnostics) {
```

Comment:
I seems that this is now only referenced by PostgreSQL resource, was intentional ?

Reply:
It's intentional, as the upgrade operation is only allowed for PostgreSQL resources for now. I opted to use `ForceNew` if user wants to update the version of and OpenSearch resource.

Reply:
There is still `version` property in MySQL resource, should we still handle that also ?

Reply:
Added `ForceNew: true` to MySQL as well. Currently there is only one whitelisted version (8), so the update issue has not arisen (yet).